core/data/src/iosMain/kotlin/io/github/droidkaigi/confsched2023/data/remoteconfig/DefaultRemoteConfigApi.kt

```diff
@@ -0,0 +1,12 @@
+package io.github.droidkaigi.confsched2023.data.remoteconfig
+
+import io.github.droidkaigi.confsched2023.data.remoteconfig.FakeRemoteConfigApi.Status.Default
+
+actual class DefaultRemoteConfigApi : RemoteConfigApi {
```

Comment:
I don't think we should make this class because iOS engineers will use this and won't notice if they use fake👀